app/src/components/features/rules/RulesListContainer/RulesTable/index.js

```diff
@@ -612,7 +612,9 @@ const RulesTable = ({
               }}
             >
               <Link
+                href={`${PATHS.RULE_EDITOR.EDIT_RULE.ABSOLUTE}/${record.id}`}
                 onClick={(e) => {
+                  e.preventDefault();
                   handleRuleNameOnClick(e, record);
```

Comment:
Hey @ncharanaraj 
The current implementation has some issues :
* Does not open in new tab using Ctrl/Cmd + click
* This rule table is also being used in shared lists page. In the rule table on shared lists page we open the rule definition in a modal instead of navigating to a different route (check the `handleRuleNameOnClick` function definition). If I **right-click** and **open the link in new tab**, this redirects to `/404`. We don't require a `href` in case of shared list rule table.